xbmc/addons/kodi-addon-dev-kit/include/kodi/General.h

```diff
@@ -727,3 +752,124 @@ inline void KodiVersion(kodi_version_t& version)
 }
 } /* namespace kodi */
 //------------------------------------------------------------------------------
+
+//==============================================================================
+namespace kodi {
```

Comment:
move curly bracket into a new line

Reply:
Normally I would say yes and no problem, but I see it a little different with the addon heads.

Once the lowercase namespace to keep apart between Kodi page and addon page.
On the other hand, the `namespace kodi {` is added to each function, so when looking over the header an outsider sees directly what is needed.

If the namespace were just like in Kodi at the beginning of a file, the linebreak would make sense, but not there for me! If a line break is added to each namespace of a function or structure, the number of lines increases unnecessarily.

Reply:
Why is the `kodi` namespace closed and opened immediately afterwards again?
Please do use custom code style, without adding it to code guidelines and clang-format.

Reply:
One to see that it is an independent function and not within a class. The huge documentary can be damn long and could be overlooked.

The other is that under circumstances a "C" interface structure could come in between which should not be in a namespace.

In order to have the addon interface as inline it is not always possible to use the guidelines correctly.